tests/functional/modules/test_zos_operator_func.py

```diff
@@ -49,8 +49,18 @@ def test_zos_operator_invalid_command(ansible_zos_module):
     hosts = ansible_zos_module
     results = hosts.all.zos_operator(cmd="invalid,command", verbose=False)
     for result in results.contacted.values():
-        assert result.get("changed") is False
-        assert result.get("exception") is not None
+        assert result.get("changed") is True
```

Comment:
Just need to double-check: you want this test to show changed is True when processing an invalid command?

Reply:
Yes because in the rc is 0 even with that command, even if bring in the message error words. Because that's the solution suggested in the conversation to be transparent with the result. Maybe I could change part of the code if bring that error keywords to keep changed as False, but be transparent with the response.

Reply:
okay, that makes sense.

Reply:
Makes sense, this is because we are transparent now and simply return back and no longer scan for keywords.